android/kroll-apt/src/main/java/org/appcelerator/kroll/annotations/generator/KrollJSONGenerator.java

```diff
@@ -509,7 +507,7 @@ protected void visitDynamicProperty(AnnotationMirror annotation, ExecutableEleme
 			Map<Object, Object> dynamicProperty = new HashMap<>(params);
 
 			String methodName = utils.getName(element);
-			String defaultName = new String(methodName);
+			String defaultName = methodName;
```

Comment:
Wondering why this was done in the first place …

Reply:
don't know :) Git blame says it has been there for 11 years :rofl:

Reply:
Hi @marshall! We missed you during at the Appc alumni meeting last week :) Sorry for dragging you into this, but do you know by any chance if there was a specific reason using the `String` constructor here? I know it's been super long, so I don't expect an answer. Thanks in any case!

Reply:
there is definitely no accounting for this spurious line of code.

my best guess.. a naive attempt to clone by Value??

sorry I missed the get together, I would have loved to be there, but I live in TX :))

Reply:
Thanks! Then I think it's good to go (since it was even recommended by Android Studio). Regarding the get together: Next time!